### PR TITLE
e2e: Test case for the etcdctlv3 'role grant' command.

### PR DESCRIPTION
**Pull request by ajityagaty**
Adding a test case to test the 'role grant' sub-command.

### CODE REVIEW

File: e2e/ctl_v3_role_test.go [hunk 1]
```diff
@@ -46,6 +50,45 @@ func roleAddTest(cx ctlCtx) {
...
+		// Try granting invalid permission to the role.
+		{
+			args:        []string{"grant", "root", "123", "foo"},
+			expectedStr: "invalid permission type",
+		},
+	}
+
+	for i, cmd := range cmdSet {
+		if err := ctlV3Role(cx, cmd.args, cmd.expectedStr); err != nil {
+			cx.t.Fatalf("roleAddTest #%d: ctlV3Role error (%v)", i, err)
```
Comment (gyuho) [inline e2e/ctl_v3_role_test.go]: typo `roleGrantTest`?

Comment (gyuho) [inline e2e/ctl_v3_role_test.go]: Or we can just `cx.t.Fatal(err)`

Comment (ajityagaty) [inline e2e/ctl_v3_role_test.go]: Yes, it's a copy paste error. Will fix it.

**Comment by gyuho** [comment]
lgtm. Will merge after CI passes. Thanks a lot!

**Comment by gyuho** [comment]
@ajityagaty *If you would like*, do you want to talk with @mitake to implement `auth disable` and help with other `auth` features?

**Comment by ajityagaty** [comment]
@gyuho Thanks ! I would be more than happy to help :)

@mitake please let me know if I can take up something.

**Comment by gyuho** [comment]
Maybe @mitake can create related issues to share and discuss his progress. Thanks all.

**Comment by ajityagaty** [comment]
@mitake Can I start looking into `auth disable` command? Would it suffice to perform an `UnsafeDelete` operation of `enableFlagKey` in order to disable auth?

**Comment by mitake** [comment]
@ajityagaty I'm planning to implement with updating the key to 0. But could you wait for a while? I have some ongoing work that would conflict with the change https://github.com/mitake/etcd/tree/auth-v3-authenticate https://github.com/mitake/etcd/tree/auth-v3-set-keys 
I think the priority of `auth disable` command isn't so high.

**Comment by ajityagaty** [comment]
@mitake Sure, I can wait ! Let me know if there is anything else that I can help with.

**Pull request merged**